components/radio/src/index.js

```diff
@@ -12,6 +11,7 @@ import {
   SPACING_POINTS,
 } from '@govuk-react/constants';
 
+const FOCUS_COLOUR = "#A89200"
```

Comment:
Instead of setting this manually, I think the govuk-colours package could just be updated from 1.0.3 to 1.1.0 (https://github.com/penx/govuk-colours/releases), and it should incorporate the new colours as per 
https://designnotes.blog.gov.uk/2019/07/29/weve-updated-the-gov-uk-colours-and-font/

Reply:
The new focus indicator govuk-colours still do not meet the required colour contrast ratio of 3:1 which is what the FSA were complaining about which is why I changed it here

Reply:
The updated govuk-colours package uses the latest GDS approved accessible list of colours, so I think that is probably the best way forward.

Reply:
base has been updated to use latest GDS colours so this change should no longer be required.